examples/functions.ejs

```diff
@@ -1,7 +1,7 @@
 <h1>Users</h1>
 
 <% function user(user) { %>
-  <li><strong><%= user.name %></strong> is a <%= user.age %> year old <%= user.species %>.</li>
+  <li><strong><%= user.name %></strong> is a <%= user.age %> year old <% echo (user.species) %>.</li>
```

Comment:
Since `echo` is a function, there shouldn't be a space between the function name and the parens. Should look like `echo(user.species);`.

Also, rather than having this example mixed together with the output tag, might it not make sense to have a totally separate example?